### ISSUE TITLE
lein.ps1 requires PowerShell 4

### ISSUE THREAD

**Issue by jevad** [technomancy/leiningen#2193]
I am recommending that you update the installation instructions to add PowerShell 4 as a requirement for Windows installations.

Trying to run the PowerShell script on Windows 7, I received an error message informing me that "Invoke-WebRequest" was not recognized.  

Research indicated that Invoke-WebRequest requires PowerShell 4 ( cf. https://matt40k.uk/2014/07/the-term-invoke-webrequest-is-not-recognized/ ).  

The default PowerShell for Windows 7 (which I think is still the version of Windows most-widely-used in industry) is PowerShell 3.

I updated to PowerShell 4, and everything I tried seems to be working well.

Microsoft's instructions for installing PowerShell 4:  http://social.technet.microsoft.com/wiki/contents/articles/21016.how-to-install-windows-powershell-4-0.aspx

**Comment by hyPiRion** [comment]
Yeah, I had the same issue, and resolved it with by installing WMF4.0. It should error out instead if you don't have a high enough version of PS.

**Comment by hyPiRion** [comment]
Actually, I'm pretty sure we can use lower level tools to support older versions of powershell. If that's possible, then I'd prefer that over powershell erroring out.

**Comment by JulianBirch** [comment]
You might find this useful. I wrote it four+ years ago, so don't expect it to have all the bells and whistles of a modern lein command. But it definitely worked on very old versions of powershell.

https://gist.github.com/395038

**Comment by hyPiRion** [comment]
@JulianBirch: Thanks, that's really useful :)
